deltachat-ffi/deltachat.h

```diff
@@ -441,7 +441,53 @@ char*           dc_get_info                  (dc_context_t* context);
  */
 char*           dc_get_oauth2_url            (dc_context_t* context, const char* addr, const char* redirect_uri);
 
+/**
+ * Get the provider information from our provider overview as json object.
+ */
```

Comment:
It returns a char pointer.  Should it be freed?  The API docs should say this.

Reply:
If the C string doesn't use the static rust string data directly (if it copies it) you must free it I guess.

Reply:
Yes, everything that is returned must be freed currently (we could change this for most of the functions though). Should I add this hint to every method or just once at the beginning?

Reply:
Sorry, all strings must be freed, but not the dc_provider_t. (currently)